shared-ts/utils/dynamo.ts

```diff
@@ -219,3 +219,52 @@ export const getAllOrganisationsInfoAndStats = async (logger: Logger): Promise<O
         throw e;
     }
 };
+
+export const getOrganisationInfoAndStats = async (
+    orgId: string,
+    logger: Logger,
+): Promise<OrganisationsWithStats[0] | null> => {
+    logger.info(`Getting organisation ${orgId} with stats from DynamoDB table...`);
+    try {
+        const dbDataInfo = await recursiveScan(
+            {
+                TableName: organisationsTableName,
+                FilterExpression: "(begins_with(SK, :info) OR begins_with(SK, :stat)) AND PK=:orgId",
+                ExpressionAttributeValues: {
+                    ":info": "INFO",
+                    ":stat": "STAT",
+                    ":orgId": orgId,
+                },
+            },
+            logger,
+        );
+
+        const orgIds = [...new Set(dbDataInfo.map((item) => item.PK))];
```

Comment:
We don't need to collect the org ids here since there's only the one, we can just have:

```ts
const info = dbDataInfo.find((item) => item.SK === "INFO");
const stats = dbDataInfo.find((item) => item.SK === "STAT");

const parsedOrgWithStats = organisationSchemaWithStats.safeParse({
    ...info,
    stats
});
```

We'd need a new schema for this:
```ts
export const organisationSchemaWithStats = organisationSchema.and(z.object({ stats: statisticSchema }));
```